zephyr/esp32/src/bt/esp_bt_adapter.c

```diff
@@ -389,12 +390,20 @@ static void intr_on(unsigned int mask)
 
 static void IRAM_ATTR interrupt_disable(void)
 {	
-	global_int_lock = irq_lock();
+	if (global_nested_counter == 0) 
+		global_int_lock = irq_lock();
```

Comment:
Not mandatory here, but should we keep this (and other occurrences) enclosed in curly braces to be consistent with Zephyr code style?

Remember we have agreed on using C89-style comments on hal_espressif the same way we do on Zephyr. What about extending that rule?

Reply:
yes!